### ISSUE TITLE
Durations are incorrect

### ISSUE THREAD

**Issue by LeaVerou** [haystack/eyebrowse-chrome-ext#29]
Eyebrowse consistently grossly underestimates the time I spend on pages. E.g. I read a document for 10 minutes then when at some point I check eyebrowse it’s listed as "a few seconds".

**Comment by amyxzhang** [comment]
Yes this is a big problem due to our current logging system being a bit of
a hack in some parts. We can look to tipsy's system and see if we can
import some of their code if they are more reliable.
https://github.com/haystack/tipsy

On Tue, Apr 28, 2015 at 4:22 AM, Lea Verou <notifications@github.com> wrote:

> Eyebrowse consistently grossly underestimates the time I spend on pages.
> E.g. I read a document for 10 minutes then when at some point I check
> eyebrowse it’s listed as "a few seconds".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/haystack/eyebrowse-chrome-ext/issues/29>.
>



-- 
Amy X. Zhang | Ph.D. student at MIT CSAIL | http://people.csail.mit.edu/axz
| @amyxzh
